shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -78,7 +78,9 @@
           .hasArg(true)
           .numberOfArgs(1)
           .argName("hosts")
-          .desc("A list of worker hosts separated by comma")
+          .desc("A list of worker hosts separated by comma, when host and locality lists are empty,"
+              + " all hosts will be selected unless excluded by setting excluded-hosts or"
```

Comment:
Can you help add one line to all the hosts related options:
Only one of the 'hosts', 'host-file', 'excluded-hosts', and 'excluded-host-file' should be set.